DecipherNow/gm-ui-components
# chore: update deps

- updated most dependencies
- refactored storybook stories, storybook config, storybook addons, and webpack config to fix breaking changes
- refactored the entry point `src/components/index.js` due to upgrading webpack.

To test 
- `npm start`, click through all stories
- `npm run build` and view the report
- follow instructions in the contributing guide to `npm link` the lib to the dashboard and make sure nothing breaks
